haproxy/entrypoint.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 set -e
-# Make sure service is running
-service rsyslog start
```

Comment:
TODO: revert

Reply:
reverted, sorry. I had this change locally to test everything works together and committed it accidentally.